openedx/core/djangoapps/user_api/accounts/serializers.py

```diff
@@ -63,6 +68,16 @@ def to_representation(self, user):
         :return: Dict serialized account
         """
         profile = user.profile
+        badges = settings.FEATURES.get('ENABLE_OPENBADGES') or False
+        if 'badges' in self.excluded_fields:
+            # Avoid doing this potentially expensive query-- it will be filtered out anyway.
```

Comment:
@talbs @antoviaque @nasthagiri The current UX compositions do not have anything for pagination. It will take a while to modify the code to allow this. I don't think we'll have it completed by this week, accordingly, though it should be possible to finish it next week.